captum/attr/_core/lrp.py

```diff
@@ -222,19 +214,12 @@ def compute_convergence_delta(self, attributions, output):
             *tensor*:
             - **delta** Difference of relevance in output layer and input layer.
         """
-
-        def _attribution_delta(attributions, output):
-            remaining_dims = tuple(range(1, len(attributions.shape)))
-            sum_attributions = torch.sum(attributions, dim=remaining_dims)
-            delta = output - sum_attributions
-            return delta
-
         if isinstance(attributions, tuple):
-            stacked_attributions = torch.stack(attributions, dim=1)
-            delta = _attribution_delta(stacked_attributions, output)
+            for attr in attributions:
+                summed_attr = sum(_sum_rows(attr) for attr in attributions)
         else:
-            delta = _attribution_delta(attributions, output)
-        return delta
+            summed_attr = _sum_rows(attributions)
+        return output.flatten() - summed_attr.flatten()
```

Comment:
I think the difference is that the GradientAttribution implementation performs forward passes on the start and end points; in this case, since we already have the output computed, so this may be more efficient. Maybe we can refactor a bit to share this part in a separate diff?

Reply:
Yes, `compute_convergence_delta` does call forward internally. I think it is concise at this point too. We can leave it as is.